ccmlib/scylla_repository.py

```diff
@@ -567,3 +562,37 @@ def run_scylla_unified_install_script(install_dir, target_dir, package_version):
     run('''{0}/install.sh --prefix {1} --nonroot{2}'''.format(
         install_dir, target_dir, install_opt), cwd=install_dir)
     run(f'''ln -s {install_dir}/scylla/conf conf''', cwd=target_dir)
+
+
+Architecture = Literal['x86_64', 'aarch64']
+
+
+def get_manager_latest_reloc_url(branch: str = "master", architecture: Architecture = None) -> str:
+    """
+    get the latest manager relocatable version of a specific branch
+    """
+    architecture = architecture or os.environ.get('SCYLLA_ARCH', 'x86_64')
+
+    url = f"https://s3.amazonaws.com/downloads.scylladb.com/manager/relocatable/unstable/{branch}/"
+    # filter only specific architecture
+    all_packages = [tar for tar in aws_bucket_ls(url) if architecture in tar]
+
+    # return latest
+    return f'https://s3.amazonaws.com/downloads.scylladb.com/{all_packages[-1]}'
```

Comment:
nit, no need to materialize the filtered package list for picking the last one. could put something like
```py
all_packages = reversed(aws_bucket_ls(url))
return next(filter(lambda tar: architecture in tar, all_packages))
```

Reply:
`aws_bucket_ls` already materialize the whole list
filtering the first few might be quicker